testcafe/tests/photos_crud.js

```diff
@@ -1,15 +1,17 @@
-import { Selector, Role } from 'testcafe' //import testcafe function
-import { regularUser } from './helpers/roles' //import roles for login
-import { getPageUrl, TESTCAFE_PHOTOS_URL, wait } from './helpers/utils'
+import { Selector, Role, ClientFunction } from 'testcafe' //import testcafe function
```

Comment:
ClientFunction => What is that?

Reply:
Permet de renvoyer des infos du client comme l'url sur laquelle il se trouve
https://devexpress.github.io/testcafe/documentation/test-api/obtaining-data-from-the-client/#creating-client-functions

Reply:
OK mais tu t'en sers pas.